src/test/java/com/exactpro/th2/infraoperator/util/SecretConfigTests.java

```diff
@@ -24,10 +24,10 @@
 import static com.exactpro.th2.infraoperator.util.HelmReleaseUtils.generateSecretsConfig;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-public class SecretConfigTest {
+class SecretConfigTests {
```

Comment:
lets add one more test case when we pass  ${secret_value:} and  ${secret_value} (without any values) we need to make sure there will be no nullpointerexception